### PR TITLE
removed privkey hex from config dictionary

### PR DESCRIPTION
**Pull request by hackaugusto**
since we introduced a feature to print the config, it's better to remove
this from the config to avoid leaking it

### CODE REVIEW

File: raiden/tests/integration/test_recovery.py [hunk 1]
```diff
@@ -50,43 +45,13 @@ def test_recovery_happy_case(
...
-        raiden_event_handler=raiden_event_handler,
-        message_handler=message_handler,
-        discovery=app0.raiden.discovery,
-    )
-
     app0.stop()
-    del app0  # from here on the app0_restart should be used
-
-    app0_restart.start()
+    app0.get()
```
Comment (konradkonrad) [inline raiden/tests/integration/test_recovery.py]: the change in the test seems unrelated to the addressed issue, can you add your rationale to the pr description?

Comment (hackaugusto) [reply raiden/tests/integration/test_recovery.py]: The change introduced a new argument to `App`, which required me to touch all the tests which restarted apps by using a new app instance. Since I had to touch this code, I went ahead and simplified it, by removing the old pattern and using the `Runnable` methods.
